estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,34 @@
+from odoo import api, fields, models
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate Property Offer Model"
+
+    price = fields.Float()
+    partner_id = fields.Many2one("res.partner", required=True)
+    property_id = fields.Many2one("estate.property", required=True)
+    status = fields.Selection(
+        selection=[('accepted', 'Accepted'), ('refused', 'Refused')],
+        copy=False
+    )
+    validity = fields.Integer(default=7)
+    date_deadline = fields.Date(compute="_compute_date_deadline", inverse="_inverse_date_deadline")
+
+    @api.depends("validity")
+    def _compute_date_deadline(self):
+        for record in self:
+            if record.create_date:
```

Comment:
From the tutorial :
"Tip: the create_date is only filled in when the record is created, therefore you will need a fallback to prevent crashing at time of creation."